scripts/rel/cut.sh

```diff
@@ -208,11 +210,13 @@ if [ -d "${CHECKS_DIR}" ]; then
 fi
 
 generate_changelog () {
-    local CHANGES_EN_MD="changes/${TAG}-en.md" CHANGES_ZH_MD="changes/${TAG}-zh.md"
-    ./scripts/format-changelog.sh "${TAG}" "en" > "$CHANGES_EN_MD"
-    ./scripts/format-changelog.sh "${TAG}" "zh" > "$CHANGES_ZH_MD"
-    git add "$CHANGES_EN_MD" "$CHANGES_ZH_MD"
-    [ -n "$(git status -s)" ] && git commit -m "chore: Generate changelog for ${TAG}"
+    local from_tag="${LAST_TAG:-}"
+    if [[ -n $from_tag ]]; then
+        local output_dir="changes"
+        ./scripts/format-changelog.sh $PROFILE "${from_tag}" "${TAG}" $output_dir
+        git add $output_dir
+        [ -n "$(git status -s)" ] && git commit -m "chore: Generate changelog for ${TAG}"
+    fi
```

Comment:
Suggestion: print a message saying that no changelog will be generated if `LAST_TAG` is undefined, it could help understanding the behavior of the script when cutting a release.